ui/app/routes.ts

```diff
@@ -10,6 +10,10 @@ export default [
     "api/curated_inferences/count",
     "routes/api/curated_inferences/count.route.ts",
   ),
+  route(
+    "api/metrics/with_feedback",
+    "routes/api/metrics/with_feedback.route.ts",
+  ),
   route(
```

Comment:
let's call the route `api/function/:function_name/feedback_counts` and use a path parameter here.